realm-jni/src/util.cpp

```diff
@@ -378,3 +398,25 @@ JStringAccessor::JStringAccessor(JNIEnv* env, jstring str)
         m_size = out_begin - m_data.get();
     }
 }
+
+jobject NewLong(JNIEnv* env, int64_t value)
+{
+    jclass cls = env->FindClass("java/lang/Long");
```

Comment:
I believe if you use 
```C++
static jclass cls = env->FindClass("java/lang/Long");
```
jclass get cached for the next call, FindClass is expensive